src/components/proposal-card/proposal.js

```diff
@@ -103,15 +102,16 @@ class Proposal extends React.PureComponent {
             </p>
           </ShortDescr>
 
-          <ViewCta
-            small
-            reverse
-            role="link"
-            onClick={this.redirectToProposalPage}
-            data-digix="Participate-Btn"
-          >
-            {cardTranslation.view}
-          </ViewCta>
+          <Link to={`/proposals/${details.proposalId}`} onClick={this.logClickToProposalPage}>
```

Comment:
Thanks, I agree to use the correct tag instead of button. Just a minor change to this though. Since the ViewCta is a button component, let’s avoid nesting it inside <a> tag. I suggest you may extend the  ViewCta component with the Link React Router using the styled() constructor in style.js then customize the styling so it resembles the current button.

Reply:
I completely agree, i thought it was weird too. I think i saw that pattern in `src/pages/help/index.js`, but maybe `<ButtonLink>` isnt creating a button element there?

I have mostly used css modules and not so much `styled-components` before.

Cant you commit the changes you proposes? And i will take a look at them. Rohit will help me with my dev environment on monday, right now i have a quite haxy setup.

Reply:
I completely agree, i thought it was weird too. I think i saw that pattern in src/pages/help/index.js, but maybe <ButtonLink> isnt creating a button element there?

I have mostly used css modules and not so much styled-components before.

Cant you commit the changes you proposes? And i will take a look at them. Rohit will help me with my dev environment on monday, right now i have a quite haxy setup.

Reply:
Hello, i have created a draft PR, it should replaces the button to an anchor tag. Let me know if you're able to review the PR below and navigate the proposals with it. 

https://github.com/DigixGlobal/governance-ui-components/pull/374

Reply:
Cool, will take a look this evening. Will set up the project locally now